src/transformers/models/led/modeling_led.py

```diff
@@ -810,6 +834,12 @@ def forward(
             attn_weights = attn_weights.view(bsz * self.num_heads, tgt_len, src_len)
 
         attn_weights = F.softmax(attn_weights, dim=-1)
+        if layer_head_mask is not None:
+            assert layer_head_mask.size() == (
+                self.num_heads,
+            ), f"Head mask for a single layer should be of size {(self.num_heads,)}, but is {layer_head_mask.size()}"
```

Comment:
Complete nit, but those asserts are not super well formatted. Can we replace them by if and raise a proper error?
```suggestion
            if layer_head_mask.size() == self.num_heads:
                raise ValueError(f"Head mask for a single layer should be of size {(self.num_heads,)}, but is {layer_head_mask.size()}")
```

(If you do one do all of them.)

Reply:
much better - I tried to find a way to make the autoformatter be nice, but couldn't figure out. Yours is great and it reads even better.

Reply:
@sgugger Thank you very much for your suggestion, I definitely agree with this. Do you think I should create a new PR to replace this part in all the models where desired?

Reply:
Since it was a nit and not a requirement, I'd say whatever works the best for you, @stancld - thank you!

Reply:
@stas00 Thanks for a quick reply. As such, I would leave it now as it is and I will replace these assertions in a new PR once there will be `head_mask` and `decoder_head_mask` implemented for all encoder-decoder models. Just in order not to mix this change in this PR, even though it's only a minor one; if it's okay :)

Reply:
That works well, @stancld! 

Thank you for your great contribution!